.env.example

```
@@ -11,6 +11,10 @@ APACHE_PID_FILE=/var/run/apache2/apache2.pid
 APACHE_RUN_DIR=/var/run/apache2
 APACHE_RUN_GROUP=www-data
 APACHE_RUN_USER=www-data
+# Optional: Enable HTTP to HTTPS redirect
+APACHE_DISABLE_OCSP_STAPLING=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APACHE_DISABLE_OCSP_STAPLING key should go before the APACHE_DOCUMENT_ROOT key

(UnorderedKey)